internal/util/update/update_test.go

```diff
@@ -309,7 +309,6 @@ func TestCommand_Run_localPackageChanges(t *testing.T) {
 				// Update upstream to Dataset2
 				UpstreamChanges: []Content{{Data: testutil.Dataset2}},
 			}
-			//defer g.Clean()
```

Comment:
This line should not be commented in first place. Must be a typo. Please add it back.